### ISSUE TITLE
[4.0] A11y - Default-Icon main menu without description

### ISSUE THREAD

**Issue by chmst** [joomla/joomla-cms#26656]
### Steps to reproduce the issue
The Main Menu (with home) has the default icon. This icon is not visible for screenreaders.

**Comment by Quy** [comment]
Please test PR #26730<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26656">issues.joomla.org/tracker/joomla-cms/26656</a>.</sub>
